gallery/src/components/NxVulnerabilityDetails/NxVulnerabilityDetailsPage.tsx

```diff
@@ -11,10 +11,10 @@ import NxVulnerabilityDetailsExample from './NxVulnerabilityDetailsExample';
 import NxVulnerabilityDetailsThirdPartyScanExample from './NxVulnerabilityDetailsThirdPartyScanExample';
 
 const exampleSourceCode = require('./NxVulnerabilityDetailsExample?raw');
-const vulnerabilityDetailsJson = require('../../resources/vulnerabilities/vulnerabilityDetailsJson.json');
+const vulnerabilityDetailsJson = require('../../assets/vulnerabilities/vulnerabilityDetailsJson.json');
 const exampleSourceForThirdPartyScans = require('./NxVulnerabilityDetailsThirdPartyScanExample?raw');
 const thirdPartyScanVulnerabilityJson =
-  require('../../resources/vulnerabilities/thirdPartyScanVulnerabilityDetails.json');
+  require('../../assets/vulnerabilities/thirdPartyScanVulnerabilityDetails.json');
```

Comment:
Do we want to leave this like this or create a more generic `/json` dir?

Reply:
> Do we want to leave this like this or create a more generic /json dir?

I think it makes sense as-is